e2e/tests/childSolicitors_test.js

```diff
@@ -23,56 +31,84 @@ const unregisteredSolicitor = {
   },
 };
 
-let mainSolicitor;
-let alternativeSolicitor;
 let caseId;
 
 Feature('Child solicitors');
 
-async function setupScenario(I, caseViewPage) {
+async function setupScenario(I) {
   if (!caseId) {
     caseId = await I.submitNewCaseWithData(mandatoryWithMaxChildren);
-    mainSolicitor = await apiHelper.getUser(config.privateSolicitorOne);
-    mainSolicitor.organisation = 'Private solicitors';
-    alternativeSolicitor = await apiHelper.getUser(config.hillingdonLocalAuthorityUserOne);
-    alternativeSolicitor.organisation = 'Hillingdon'; // org search on aat does not like London Borough Hillingdon
   }
-  await I.navigateToCaseDetailsAs(config.hmctsAdminUser, caseId);
-  await caseViewPage.goToNewActions(config.administrationActions.amendChildren);
+  if (!solicitor1.details) {
+    solicitor1.details = await apiHelper.getUser(solicitor1);
+    solicitor1.details.organisation = 'Private solicitors';
+  }
+  if (!solicitor2.details) {
+    solicitor2.details = await apiHelper.getUser(solicitor2);
+    solicitor2.details.organisation = 'Hillingdon'; // org search on aat does not like London Borough Hillingdon
+  }
+  if (!solicitor3.details) {
+    solicitor3.details = await apiHelper.getUser(solicitor3);
+    solicitor3.details.organisation = 'Wiltshire County Council';
+  }
 }
 
-Scenario('HMCTS Confirm that a main solicitor in not assigned for all the children yet', async ({I, caseViewPage, enterChildrenEventPage}) => {
-  await setupScenario(I, caseViewPage);
+Scenario('Solicitor cannot request representation before case submission and Cafcass solicitor is set', async ({I, caseListPage, caseViewPage, submitApplicationEventPage, noticeOfChangePage}) => {
+  await setupScenario(I);
+  await I.signIn(solicitor1);
+  caseListPage.verifyCaseIsNotAccessible(caseId);
+
+  await noticeOfChangePage.navigate();
+  await noticeOfChangePage.enterCaseReference(caseId);
+  I.click('Continue');
+  I.see('Your notice of change request has not been submitted');
```

Comment:
Instead of using `I.see(...)` which does an immediate check, it might be more robust to either `I.waitForText(...)` for that line or `I.waitForElement(...)` to wait for an attribute of the error before checking for the text.
This could better handle environmental / API delays.